tsdb/block.go

```diff
@@ -453,6 +502,11 @@ func (r blockIndexReader) Close() error {
 	return nil
 }
 
+// LabelValueFor returns value of given label of metric referred to by id.
```

Comment:
Here and other comments

```suggestion
// LabelValueFor returns value of given label value of series referred to by ID.
```

Reply:
note: there are similar comments like this :)

Reply:
sorry about that, i updated the remaining ones https://github.com/prometheus/prometheus/pull/8400/commits/ed646466c4de35d44ddbb266a3342241bc997a3f